drgn/commands/_builtin/crash/vm.py

```diff
@@ -0,0 +1,233 @@
+# Copyright (c) 2025, Kylin Software, Inc. and affiliates.
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+import argparse
+from typing import Any, List
+
+from drgn import Object, Program
+from drgn.commands import CommandArgumentError, argument, drgn_argument
+from drgn.commands.crash import (
+    CrashDrgnCodeBuilder,
+    _pid_or_task,
+    crash_command,
+    crash_get_context,
+)
+from drgn.helpers.common.format import print_table
+from drgn.helpers.linux.fs import inode_path
+from drgn.helpers.linux.mm import for_each_vma, get_task_rss_info
+from drgn.helpers.linux.sched import task_cpu
+
+
+def _get_file_path(vma: Object, mm: Object) -> str:
+    """Get the file path name corresponding to the VMA"""
+    if vma.vm_file:
+        try:
+            path_bytes = inode_path(vma.vm_file.f_inode)
```

Comment:
`inode_path()` is relative to the containing filesystem. Use `d_path(vma.vm_file.f_path)` for an absolute path.